.pre-commit-config.yaml

```diff
@@ -196,9 +196,10 @@ repos:
       - id: markdown-link-check
         args: ["--quiet", "--config", ".md-link-config.json"]
   - repo: https://github.com/igorshubovych/markdownlint-cli
-    rev: v0.44.0
+    rev: v0.45.0
     hooks:
       - id: markdownlint
+        language_version: 22.14.0
```

Comment:
@cbachhuber @lukasoyen Could you check if this version works with Python 3.8 or if we have to use `20.19.2`.

Reply:
```
➜  dev-tools git:(renovate/dependency-updates) python3 --version
Python 3.8.10
➜  dev-tools git:(renovate/dependency-updates) pre-commit run --all-files markdownlint
markdownlint.............................................................Passed
```